dspace-api/src/main/java/org/dspace/app/itemimport/ItemImportServiceImpl.java

```diff
@@ -1518,28 +1676,62 @@ protected void processOptions(Context c, Item myItem, List<String> options)
             if (notfound && !isTest) {
                 // this should never happen
                 System.out.println("\tdefault permissions set for "
-                                       + bitstreamName);
+                    + bitstreamName);
             } else if (!isTest) {
                 if (permissionsExist) {
                     if (myGroup == null) {
                         System.out.println("\t" + groupName
-                                               + " not found, permissions set to default");
+                            + " not found, permissions set to default");
                     } else if (actionID == -1) {
                         System.out
                             .println("\tinvalid permissions flag, permissions set to default");
                     } else {
                         System.out.println("\tSetting special permissions for "
-                                               + bitstreamName);
+                            + bitstreamName);
                         setPermission(c, myGroup, actionID, bs);
                     }
                 }
 
                 if (descriptionExists) {
                     System.out.println("\tSetting description for "
-                                           + bitstreamName);
+                        + bitstreamName);
                     bs.setDescription(c, thisDescription);
                     bitstreamService.update(c, bs);
                 }
+
+                if (labelExists) {
+                    MetadataField metadataField = metadataFieldService
+                        .findByElement(c, "iiif", "label", null);
```

Comment:
we have constants for the iiif metadata in `org.dspace.app.iiif.service.utils.IIIFUtils` if the constants here cannot be used directly we should split in more constants and build the existing constants combining them (i.e.
 `METADATA_IMAGE_WIDTH = "iiif.image.width";` can be `METADATA_IMAGE_WIDTH = "METADATA_IIIF_SCHEMA + "." + METADATA_IMAGE_ELEMENT + "." + METADATA_IMAGE_WIDTH_QUALIFIER `;